src/components/common/ListPeopleAndRoles.vue

```diff
@@ -233,31 +218,34 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
 }
 
 .people-roles-title {
+  font-weight: bold;
   color: $gray9;
 }
 
 .people-roles-content {
   margin-top: 0.5rem;
-  padding: 0.5rem 1.25rem 0.5rem 1.25rem;
+  padding: 0.5rem 0.5rem 0.5rem 1.25rem;
   border-top: 1px solid $gray1;
   font-size: $px-14;
   color: $gray7;
 
   .actions {
-    position: absolute;
-    right: 0;
-
     .edit-action {
       border-right: 1px solid $gray1;
     }
 
-    .v-btn {
-      min-width: 0.5rem;
-    }
+    // .v-btn {
+    //   min-width: 0.5rem;
+    // }
```

Comment:
this is commented out because we might need it?

Reply:
Oops. I was trying things both ways. It's not needed -- deleted in next commit.